doc/pages/example_notebooks/sim4D/sim4D.v4.ipynb

```diff
@@ -54,7 +66,9 @@
     "sim.add(PropagationCK(Bfield))\n",
     "sim.add(FutureRedshift())\n",
     "sim.add(PhotoPionProduction(CMB()))\n",
-    "sim.add(PhotoPionProduction(IRB_Kneiske04()))\n",
+    "ppp_irb = PhotoPionProduction(IRB_Kneiske04())\n",
+    "ppp_irb.setCorrectionFactor(1)\n",
```

Comment:
Why do you manually change the correction factor? I guess this is a leftover from testing; in that case please remove it to prevent confusion. Otherwise some comment should be added.

Reply:
Yes it is only left over from testing. I changed it back to the previous version.